tutorials/scripting/gdextension/gdextension_cpp_example.rst

```diff
@@ -0,0 +1,643 @@
+.. _doc_gdextension_cpp_example:
+
+GDExtension C++ example
+====================
+
+Introduction
+------------
+
+The C++ bindings for GDExtension are built on top of the C GDExtension API
+and provide a nicer way to "extend" nodes in Godot using C++. This new system allows the 
```

Comment:
"extend" any Class

Reply:
I do not know if "any" is accurate but it's more then only the nodes.

Reply:
There is no `any`? or do you mean you want it more specific?

Reply:
I see all the demo cases extending nodes. Maybe in the tutorial, there should be some reference that other things are also able to extend.

Reply:
Updated it with this: 
> nicer way to "extend" nodes and other built-in classes in Godot using C++.